src/components/Cluster/ClusterDetail/NodePool.js

```diff
@@ -119,7 +121,27 @@ class NodePool extends Component {
         {/* Hide the rest of fields when editing name */}
         {!isNameBeingEdited && (
           <>
-            <Code>{node_spec.aws.instance_type}</Code>
+            <Code>
+              {node_spec.aws.use_alike_instance_types
+                ? 'mixed'
+                : node_spec.aws.instance_type}
+            </Code>
+            <NodesWrapper>
+              {instance_distribution &&
+              typeof instance_distribution.on_demand_base_capacity !==
+                'undefined'
+                ? instance_distribution.on_demand_base_capacity
+                : '-'}
```

Comment:
```suggestion
              {instance_distribution?.on_demand_base_capacity ?? '-'}
```

Reply:
I had this previously like this:
`instance_distribution?.on_demand_base_capacity || '-'`
This broke for value `0`. I'll test this though

Reply:
`||` will guard against any empty value (including `0`),  but `??` will only guard against nullable values (`undefined` or `null`)